article.md

```diff
@@ -541,12 +546,12 @@ utilisant des fonctions classiques présentées en début de section.
 
```

Comment:
C'est surtout `async with` et `async for` qui permettent ça. Du moins plus que précédement.